features/specs/S-en-T-test-omgekeerd-ouders/Tg033.feature

```diff
@@ -403,10 +420,12 @@ Functionaliteit: Tg033 - Paassen-Pauw - Twee gehuwde vrouwen met 4 kinderen
       | naam                | waarde    |
       | burgerservicenummer | 000000048 |
     En heeft de persoon een 'gezag' met de volgende gegevens
-      | naam                             | waarde    |
-      | type                             | Voogdij   |
-      | minderjarige.burgerservicenummer | 000000048 |
-    En heeft 'gezag' geen derden
+      | naam                             | waarde           |
+      | type                             | GezamenlijkGezag |
+      | minderjarige.burgerservicenummer | 000000048        |
+      | ouder.burgerservicenummer        | 000000012        |
+      | derde.burgerservicenummer        | 000000024        |
```

Comment:
zie opmerking hierboven. ik denk dat dit onbekende derde moet zijn want ger. uitspraak 1D

Reply:
Je hebt gelijk. Er is verwarring ontstaan wat wij verstaan onder _niet ouder_ (derde) vanuit het domein en _niet ouder_ vanuit gezag-api.

Gezag uitspraak bevat nu `OnbekendeDerde`.